docs/cookbook/infrastructure/flox-store.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Catalog Store
+description: Create a Catalog Store for publishing your own Flox packages
+---
+
+# Setting up a Catalog Store
+
+Publishing your own software to your organization's Flox Catalog requires some
+initial setup, but the process is relatively straightforward. Flox supports
+publishing packages to a Catalog Store, which will exist in an AWS S3 bucket
+that is managed by your organization. In order to use this bucket to store
+binaries built with Flox, you will need to set ingress and egress URIs on the
```

Comment:
I would say "artifacts" or "packages" since we can store more than just executables.